### ISSUE TITLE
Support others distributions

### ISSUE THREAD

**Issue by Yamakaky** [DragonComputer/Dragonfire#51]
I think a good start would be to add a `requirements.txt` with all the python dependencies and a list of native dependencies. I'm on archlinux so I can be a test subject XD

**Comment by mertyildiran** [comment]
@Yamakaky you are right, porting this virtual assistant to other distros is a very crucial task. We seek other people's(for example yours) help for this task. Feel free to add Arch Linux support and send a pull request. That would be very helpful. :blush:
